Docs for v0.46.0 failed to build

![image](https://user-images.githubusercontent.com/639336/103468150-13607680-4d0b-11eb-8ccd-e3869edf1ec1.png)

~~Looks like crates behind `ws` feature is not installed:~~

Looks like TLS features for `async-tungstenite` is not enabled:

```text
[INFO] [stderr] error[E0432]: unresolved import `async_tungstenite::tokio::connect_async_with_tls_connector`
[INFO] [stderr]   --> src/client/mod.rs:21:13
[INFO] [stderr]    |
[INFO] [stderr] 21 |     tokio::{connect_async_with_tls_connector, ConnectStream},
[INFO] [stderr]    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[INFO] [stderr]    |             |
[INFO] [stderr]    |             no `connect_async_with_tls_connector` in `tokio`
[INFO] [stderr]    |             help: a similar name exists in the module: `connect_async_with_config`
```

See https://docs.rs/crate/kube/0.46.0/builds/331723

Maybe adding `"ws-native-tls"` to the features or using `all-features = true` (#365) will fix it,

--- a/kube/Cargo.toml
+++ b/kube/Cargo.toml
@@ -27,7 +27,7 @@ ws-native-tls = ["real-native-tls", "tokio-native-tls", "async-tungstenite/tokio
 ws-rustls-tls = ["tokio-rustls", "async-tungstenite/tokio-rustls"]
 
 [package.metadata.docs.rs]
-features = ["derive", "ws"]
+features = ["derive", "ws", "ws-native-tls"]
 
 [dependencies]
 base64 = "0.12.1"

